de.bund.bfr.knime.js/src/js/app/app.editable.mt.InputForm.js

```diff
@@ -266,15 +266,19 @@
 				}
 			
             }else {
-                isValid = O.input.val() ? true : false;
+				
+				isValid = O.input.val() ? true : false;
+				if(O.type === "email"){
+					isValid = true;	
+				}
                 if(!isValid){
                     O.input.$validationContainer.text("required");
                 }
                 // check if mail has correct structure
-                if(isValid && O.type === "email"){
-                    isValid= /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/.test(O.input.val());
-                    O.input.$validationContainer.text("Not a valid email value");
-                }
+                //if(isValid && O.type === "email"){
```

Comment:
I don't think we should cancel the validation if the user provide some text.
As I understood the request was about making it optional and that can be achieved by relaxing the schema in editor_data_1.0.0.js and add the required code to validate the email text if provided.